### PR TITLE
Fix typo in the Bayes factor notebook

### LINKED ISSUE #3726: Small typo in the Bayes factor notebook.
Hi,

I found a small typo in this notebook. I guess it should be `It's` or `It is`. 

image

https://docs.pymc.io/notebooks/Bayes_factor.html

### PR DESCRIPTION
**Pull request by ac-optimus**
This will fix #3726

### CODE REVIEW

File: docs/source/notebooks/SMC2_gaussians.ipynb [hunk 1]
```diff
@@ -49,7 +49,7 @@
...
     "\n",
-    "The previous paragraph is summarized in the next figure, the first subplot show 5 samples (orange dots) at some particular stage. The second subplots show how this samples are reweighted according to the their posterior density (blue Gaussian curve). The third subplot shows the result of running a certain number of Metropolis steps, starting from the _selected/reweighting_ samples in the second subplots, notice how the two samples with the lower posterior density (smaller circles) are discarded and not used to seed Markov chains.\n",
+    "The previous paragraph is summarized in the next figure, the first subplot show 5 samples (orange dots) at some particular stage. The second subplots show how this samples are reweighted according to their posterior density (blue Gaussian curve). The third subplot shows the result of running a certain number of Metropolis steps, starting from the _selected/reweighting_ samples in the second subplots, notice how the two samples with the lower posterior density (smaller circles) are discarded and not used to seed Markov chains.\n",
```
Comment (aloctavodia) [inline docs/source/notebooks/SMC2_gaussians.ipynb]: suggested change 

The previous paragraph is summarized in the next figure, the first subplot shows 5 samples (orange dots) at some particular stage. The second subplot shows how these samples are reweighted according to their posterior density (blue Gaussian curve). The third subplot shows the result of running a certain number of Metropolis steps, starting from the *selected/reweighting* samples in the second subplot, notice how the two samples with the lower posterior density (smaller circles) are discarded and not used to seed Markov chains.

**Comment by aloctavodia** [comment]
Hi @ac-optimus, thanks for your contribution. 

Notice that I changed the title of the PR. For a future reference try to avoid expressions like "fix issue #xxxx" and instead use more informative sentences.

Also check this super nice [github feature](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords)

I also noticed that a commit from #3770 has been leaked here. Always remember to use branches and never work from your master branch. This will help to avoid a lot of headaches, I can tell you that from my own experience, haha.

**Comment by ac-optimus** [comment]
Hi @aloctavodia, I have added the suggested changes. Thanks for correcting the title and sorry for the silly mistakes😅, will surely follow these advises from next time. How about I create new PRs with both the issue fixes in different branch?

Thanks!

**Comment by aloctavodia** [comment]
No problem, we all have made the same or similar silly mistakes at some point in the past (and surely we will do some more in the future). No need for a new PR. I think you can rebase and revert the leaked commit in this PR.

**Comment by ac-optimus** [comment]
Hi, done that! Have a look.

**Comment by aloctavodia** [comment]
Thanks @ac-optimus!

**Pull request merged**